### PR TITLE
Adding basic Ophan ad tracking to commercial

### PR DESCRIPTION
**Pull request by kenlim**
This used to be done in Ophan’s Tracker JS, but the switch to the new Curl/JSPM dependency loading has broken it. So now I'm moving it in here.

### CODE REVIEW

**Comment by uplne** [comment]
:+1:

**Comment by OliverJAsh** [comment]
How did the migration break it? BTW we're still using curl, we haven't finished the migration yet.

File: static/src/javascripts/projects/common/utils/detect.js [hunk 1]
```diff
@@ -109,6 +109,15 @@ define([
         return totalTime;
     }
 
+    function getTimeOfDomComplete(performance) {
```
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/utils/detect.js]: This argument is never passed in?

File: static/src/javascripts/projects/common/utils/detect.js [hunk 2]
```diff
@@ -109,6 +109,15 @@ define([
...
+        var perf = performance || window.performance || window.msPerformance || window.webkitPerformance || window.mozPerformance;
```
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/utils/detect.js]: Could we define this at the module level and then re-use it inside each fn?

**Comment by kenlim** [comment]
Ophan was getting imported too early, so it tried to work with an "Undefined" object.

File: static/src/javascripts/projects/common/utils/detect.js (hunk 1 above)
Comment (kenlim) [inline static/src/javascripts/projects/common/utils/detect.js]: Ah, yes, that's for testing.

File: static/src/javascripts/projects/common/utils/detect.js (hunk 2 above)
Comment (kenlim) [inline static/src/javascripts/projects/common/utils/detect.js]: Yes, we should have a conversation about some sort of standardised object to expose this data.

**Pull request merged**